src/adagrams.js

```diff
@@ -1,15 +1,170 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+import _, { map } from "underscore";
```

Comment:
You don't need to import `map` since you're only using the built-in `map` method on arrays:

```suggestion
import _ from "underscore";
```